<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newVpnConSettingWindow</class>
 <widget class="QMainWindow" name="newVpnConSettingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_1" stretch="2,4,4,3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,8">
        <item>
         <widget class="QLabel" name="labelIcon_1">
          <property name="text">
           <string>Icon</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1">
          <item>
           <widget class="QLabel" name="labelTitle_1">
            <property name="text">
             <string>Please configure the VPN Connection Setting for VPN Server.</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEditSettingName">
              <property name="text">
               <string>New VPN Connection</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="labelSettingName">
              <property name="text">
               <string>SettingName</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxDestVpnServer">
        <property name="title">
         <string>Destination VPN Server:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,3">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="labelIcon_2">
             <property name="text">
              <string>Icon</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelTitle_2">
             <property name="text">
              <string>Specify the hostname or IP address, and the port number
and the Virtual Hub on thedestination VPN Server</string>
             </property>
             <property name="textFormat">
              <enum>Qt::AutoText</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="2" column="0">
            <widget class="QLabel" name="labelHubName">
             <property name="text">
              <string>Virtual Hub Name</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="labelHostName">
             <property name="text">
              <string>Host Name</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelPortNumber">
             <property name="text">
              <string>Port Number</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEditHostName"/>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="comboBoxHubName">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,2">
             <item>
              <widget class="QComboBox" name="comboBoxPort">
               <property name="editable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBoxDisNATT">
               <property name="text">
                <string>Disable NAT-T</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxProxyServer">
        <property name="title">
         <string>Proxy Server as Relay:</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,7">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="labelIcon_3">
           <property name="text">
            <string>Icon</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="labelTitle_3">
             <property name="text">
              <string>You can connect to a VPN Server via proxy server.</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_3" columnstretch="2,5" columnminimumwidth="0,0">
             <item row="1" column="1">
              <widget class="QRadioButton" name="radioButtonConnectTcpIp">
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Direct TCP/IP Connection (No Proxy)</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QRadioButton" name="radioButtonConnectHttp">
               <property name="text">
                <string>Convert via HTTP Proxy Server</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="pushButtonImportProxyServer">
               <property name="text">
                <string>Import IE Proxy Server</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="labelProxyType">
               <property name="text">
                <string>Proxy Type</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QRadioButton" name="radioButtonConnectSocks">
               <property name="text">
                <string>Convert via SOCKS Proxy Server</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QPushButton" name="pushButtonProxyServerSetting">
               <property name="text">
                <string>Proxy Server Settings</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxServerCertOptions">
        <property name="title">
         <string>Server Certification Option:</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,7">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="labelIcon_4">
           <property name="text">
            <string>Icon</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QCheckBox" name="checkBoxAlwaylVerifyServCert">
             <property name="text">
              <string>Always Verify Server Certificate</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonManageCert">
             <property name="text">
              <string>Manage Trusted CA Certificate List</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QPushButton" name="pushButtonIndividualCert">
               <property name="text">
                <string>Cpecify Individual Cert</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonShowIndividualCert">
               <property name="text">
                <string>Show Individual Cert</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="4,5,5,1">
      <item>
       <widget class="QGroupBox" name="groupBoxNic">
        <property name="title">
         <string>Virtual Network Adapter to Use:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QListWidget" name="listWidgetNic"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxAdvancedSetting">
        <property name="title">
         <string>User Authentication Setting:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8" stretch="2,3,4">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,8">
           <item>
            <widget class="QLabel" name="labelIcon5">
             <property name="text">
              <string>Icon</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelTitle_4">
             <property name="text">
              <string>Set the user authentication information that is required when
connecting to the VPN Server:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <property name="verticalSpacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>50</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>50</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="labelTypeAuth">
             <property name="text">
              <string>Auth Type</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelUserName">
             <property name="text">
              <string>User Name</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="comboBoxTypeAuth"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEditUserName"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayoutCustom">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="2,7">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="labelIcon">
               <property name="text">
                <string>Icon</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelTitle_5">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="pushButtonViewCertificate">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonSpecifyCertificate">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxAuthenticationSetting">
        <property name="title">
         <string>GroupBox</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>140</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonOk">
          <property name="text">
           <string>Ok</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonCancel">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew_VPN_Connection_Setting_Properties">
    <property name="title">
     <string>New VPN Connection Setting Properties</string>
    </property>
   </widget>
   <addaction name="menuNew_VPN_Connection_Setting_Properties"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
